frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -77,36 +83,50 @@ const getColumns = () => {
   ];
 };
 
+const isNodeUnschedulable = (node: NodeKind) => {
```

Comment:
`hasTaints` would be a better name because this sounds like it's checking`spec.unschedulable`.

```suggestion
const hasTaints = (node: NodeKind) => {
```